### DISCUSSION TITLE
Github Copilot doesn't work after the word "killed" (sometimes)

### DISCUSSION THREAD

**Discussion by KeaneCarotenuto** [community/community#32855] [Copilot Conversations]
After using it perfectly fine for hours on the same project, it stops suggesting after one specific line of code in one file only.

Working fine before problem line:
![image](https://user-images.githubusercontent.com/16412361/189813168-fc175200-c339-4ddd-a260-566dfc6d1f5b.png)

Stops working after problem line (line 24 here)
![image](https://user-images.githubusercontent.com/16412361/189813053-7ecf08c7-4065-45ff-89ca-ae05166950ce.png)

Removing problem line, it starts working again (line 24 in prev pic gone)
![image](https://user-images.githubusercontent.com/16412361/189813090-66d83c1b-1c59-4091-94e9-663badd8bab0.png)

**EDIT:** Changing the words "killed" to anything else (or deleting them) makes it work perfectly fine
![image](https://user-images.githubusercontent.com/16412361/189814078-86d70f7e-966c-4765-9a7e-beab91aeb813.png)

I have tried:
- Disabling/Enabling CP, then reloading VSCode.
- It works in literally any other file.
- It works on new files.
- Reverting/Updating CP, then reloading VSCode.
- Opening VSCode in admin.

Any help/explanation would be appreciated.

**EDIT:**
Github CP refuses to work with the word "killed" in it??
Excuse me, and I being censored by an AI here? (just a joke, but what is actually going on here?)

**EDIT 2:**
It seems like Github Copilot does actually prevent suggestions if it detects "bad" words/topics, as per this 2021 discussion.
https://github.com/community/community/discussions/7079

It would be great if there was any kind of user feedback at all about this...

**Comment by iamcjwilliams** [comment]
Hello @KeaneCarotenuto Our engineering team is looking into this issue
